index.js

```diff
@@ -77,6 +77,12 @@ Prober.prototype.notok = function notok() {
     }
 };
 
+Prober.prototype.setEnabled = function setEnabled(enabled) {
+    if (typeof enabled === 'boolean') {
```

Comment:
it seems bad to fail silently. I'm ok with not checking this condition and do `!!enabled`